pkg/instrumentation/dotnet.go

```diff
@@ -28,17 +28,32 @@ const (
 	dotNetOTelAutoHomePath              = "/otel-auto-instrumentation-dotnet"
 	dotNetSharedStorePath               = "/otel-auto-instrumentation-dotnet/store"
 	dotNetStartupHookPath               = "/otel-auto-instrumentation-dotnet/net/OpenTelemetry.AutoInstrumentation.StartupHook.dll"
+	dotNetAutoPlugins                   = "AWS.Distro.OpenTelemetry.AutoInstrumentation.Plugin, AWS.Distro.OpenTelemetry.AutoInstrumentation"
 	dotnetVolumeName                    = volumeName + "-dotnet"
 	dotnetInitContainerName             = initContainerName + "-dotnet"
 	dotnetInstrMountPath                = "/otel-auto-instrumentation-dotnet"
 )
 
+const (
+	dotNetCoreClrProfilerPathWindows = "C:\\otel-auto-instrumentation-dotnet\\win-x64\\OpenTelemetry.AutoInstrumentation.Native.dll"
+	dotNetAdditionalDepsPathWindows  = "C:\\otel-auto-instrumentation-dotnet\\AdditionalDeps"
+	dotNetOTelAutoHomePathWindows    = "C:\\otel-auto-instrumentation-dotnet"
+	dotNetSharedStorePathWindows     = "C:\\otel-auto-instrumentation-dotnet\\store"
+	dotNetStartupHookPathWindows     = "C:\\otel-auto-instrumentation-dotnet\\net\\OpenTelemetry.AutoInstrumentation.StartupHook.dll"
+	dotnetInstrMountPathWindows      = "\\otel-auto-instrumentation-dotnet"
+)
+
 // Supported .NET runtime identifiers (https://learn.microsoft.com/en-us/dotnet/core/rid-catalog), can be set by instrumentation.opentelemetry.io/inject-dotnet.
 const (
 	dotNetRuntimeLinuxGlibc = "linux-x64"
 	dotNetRuntimeLinuxMusl  = "linux-musl-x64"
```

Comment:
Will CORECLR_PROFILER_PATH be set for "linux-musl-x64" like [OTel Operator](
https://github.com/open-telemetry/opentelemetry-operator/blob/0cafedf0c150a3f7c48842e6b3e5f535385737f7/tests/e2e-instrumentation/instrumentation-dotnet-musl/01-assert.yaml#L27)?

```
    - name: CORECLR_PROFILER_PATH
      value: /otel-auto-instrumentation-dotnet/linux-musl-x64/OpenTelemetry.AutoInstrumentation.Native.so
```